src/Symfony/Component/CssSelector/Parser/Parser.php

```diff
@@ -257,6 +265,32 @@ private function parseSimpleSelector(TokenStream $stream, bool $insideNegation =
         return [$result, $pseudoElement];
     }
 
+    private function parseSimpleSelectorArguments(TokenStream $stream): array
+    {
+        $arguments = [];
+        while (true) {
+            [$result, $pseudoElement] = $this->parseSimpleSelector($stream, true);
```

Comment:
The naming in this parser is confusing. `parseSimpleSelector` is not parsing a simple selector but a compound selector. However, this is still wrong for the parsing of `is` and `where`. They take a selector list, i.e. a list of *complex selectors*, not of *compound selectors* (see https://www.w3.org/TR/selectors-4/#structure for the terminology)

Reply:
This method has been remove since we use `parseSelectorList` instead.